Makefile

```diff
@@ -10,6 +11,11 @@ install-carthage:
 	brew remove carthage --force || true
 	brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/af232506f5f1879af77852d6297b1e2a5b040270/Formula/carthage.rb
 
+install-bazel-%:
```

Comment:
Where does this get called from?

Reply:
https://github.com/lyft/mapper/blob/1a6d689cba3726fc400a16e59db6da299b3ca2ad/.travis.yml#L22